app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+import os
+import shutil
+
+
+def make_directories(destination_path: str) -> None:
+
+    directories = destination_path.split(os.path.sep)
+    current_path = ""
+
+    for directory in directories:
+        current_path = os.path.join(current_path, directory)
+
+        if not os.path.exists(current_path):
+            os.makedirs(current_path, exist_ok=True)
```

Comment:
os.makedirs(destination_path, exist_ok=True). This single line of code accomplishes the same task as the code you provided without the need to split the path and loop through the directories